src/containers/Event.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {IChangeEvent} from 'react-jsonschema-form';
+import {EditableForm} from '../lib/cms/components/EditableForm';
+import {isUserAuthorized} from '../utils/auth';
+import {ICMSData} from '../models';
+
+interface IEventContext {
+  cmsData: ICMSData // CMS data to be rendered
+  onCmsDataUpdate: (data: any, path: string[]) => void // Callback to execute when a particular CMS
+                                                              // data needs to be updated
+}
+
+export class Event extends React.Component<any, any> {
+
+  context: IEventContext;
+
+  static contextTypes: React.ValidationMap<any> = {
```

Comment:
Yes we do, not entirely sure why, but it won't run unless I specify the contextTypes.